src/edu/csus/ecs/pc2/ui/EditRunPane.java

```diff
@@ -270,7 +270,35 @@ protected void updateRun() {
 
             RunStates prevState = run.getStatus();
             RunStates newRunState = (Run.RunStates) runStatusComboBox.getSelectedItem();
-
+            String errMsg = null;
+            
+            // Make sure it's safe to change the runstate
+            switch(newRunState) {
+            case JUDGED:
+            case MANUAL_REVIEW:
+            case REJUDGE:           // TODO: remove REJUDGE state completely since nobody appears to use it -- JB
```

Comment:
@johnbrvc I don't understand the above comment; I've seen multiple contests where judges "rejudged" a run (or many runs...)

Reply:
Of course judges rejudge (I do it all the time), but that particular STATE is never used.  "BEING_RE_JUDGED" is the state that is used.  I think that REJUDGE must have been a left-over or something.  Hence, my comment.